cmd/cosign/cli/verify/verify_blob_test.go

```diff
@@ -957,6 +955,166 @@ func TestVerifyBlobCmdWithBundle(t *testing.T) {
 			t.Fatalf("expected error with mismatched issuer, got %v", err)
 		}
 	})
+	t.Run("Intermediate root not explicit in non-experimental mode", func(t *testing.T) {
```

Comment:
nit: `s/not explicit/from bundle/`?

Reply:
Hmm no: the intermediate is not in the bundle. Changed to:

`Non-explicit Fulcio roots with bundle in non-experimental mode`

Reply:
hahaha implciit = non-explicit